public/docs/ts/latest/tutorial/toh-pt5.jade

```diff
@@ -83,57 +105,55 @@ code-example(language="bash").
 
   Our current app loads `AppComponent` and immediately displays the list of heroes.
 
-  Our revised app should present a shell with a choice of views (*Dashboard* and *Heroes*) and then default to one of them.
+  Our revised app should present a shell with a choice of views (*Dashboard* and *Heroes*) 
+  and then default to one of them.
 
   The `AppComponent` should only handle navigation.
   Let's move the display of *Heroes* out of `AppComponent` and into its own `HeroesComponent`.
 
   ### *HeroesComponent*
+
   `AppComponent` is already dedicated to *Heroes*.
   Instead of moving anything out of `AppComponent`, we'll just rename it `HeroesComponent`
   and create a new `AppComponent` shell separately.
 
   The steps are to rename:
-  * `app.component.ts` file to `heroes.component.ts`
+  * <span ngio-ex>app.component.ts</span> file to <span ngio-ex>heroes.component.ts</span>
   * `AppComponent` class to `HeroesComponent`
   * Selector `my-app` to `my-heroes`
 
-:marked
-+makeExample('toh-5/ts/app/heroes.component.ts', 'heroes-component-renaming', 'app/heroes.component.ts (showing renamings only)')(format=".")
++makeExcerpt('app/heroes.component.ts (showing renamings only)', 'renaming')
 
 :marked
   ## Create *AppComponent*
+
   The new `AppComponent` will be the application shell.
   It will have some navigation links at the top and a display area below for the pages we navigate to.
 
   The initial steps are:
 
-  * create a new file named `app.component.ts`.
-  * define an `AppComponent` class.
-  * `export` it so we can reference it during bootstrapping in `app.module.ts`.
-  * expose an application `title` property.
-  * add the `@Component` metadata decorator above the class with a `my-app` selector.
-  * add a template with `<h1>` tags surrounding a binding to the `title` property.
-  * add the `<my-heroes>` tags to the template so we still see the heroes.
-  * add the `HeroesComponent` to the root NgModule's `declarations` array so Angular recognizes the `<my-heroes>` tags.
-  * add the `HeroService` to the root NgModule's `providers` array because we'll need it in every other view.
-  * add the supporting `import` statements.
+  * Create the file <span ngio-ex>app/app.component.ts</span>.
+  * Define an <span if-docs="ts">exported</span> `AppComponent` class.
+  * Add an `@Component` !{_decorator} above the class with a `my-app` selector.
+  * Move from `HeroesComponent` to `AppComponent` the
```

Comment:
We don't use ", and" in bulleted lists, so how about:

* Move the following from `HeroesComponent` to `AppComponent`:
  * `title` class property
  * `@Component` template's `<h1>` element, which contains a binding to `title`